### DISCUSSION TITLE
单独的服务读取nacos 上对应的服务配置文件这个逻辑是怎么实现的？

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#257] [02.提问]
单独的服务读取nacos 上对应的服务配置文件这个逻辑是怎么实现的？
这个实现原理是什么？ 有地方配置吗？

**Comment by zuihou** [comment]
spring-cloud 自己实现的，用就行了。不明白就百度 spring-cloud 如何读取nacos或spring-config中的配置
